### ISSUE TITLE
Use of IntEnum in tasks/models 

### ISSUE THREAD

**Issue by dybi** [golemfactory/concent#425]
Right now we are using different type of enums (`Enum`, `IntEnum`, `ChoiceEnum`, etc.). The problem is, that enums are often passed to celery tasks. And everything that comes as argument to celery task should be JSON serializable. From all mentioned enums, only `IntEnum` is JSON serializable (see: [example on StackOverflow](https://stackoverflow.com/questions/20826725/django-celery-send-register-email-do-not-work)  ) so it should be used throughout the code.

**TL;DR**
Change all enums in `models` and `tasks` to be `IntEnum`.

**Comment by cameel** [comment]
@dybi @pawelkisielewicz This is no longer relevant and should be closed, right? Could you summarize what we ended up doing?
